x/oracle/types/ballot.go

```diff
@@ -146,11 +146,22 @@ func NewClaim(power, mandatoryWinCount int64, recipient sdk.ValAddress) Claim {
 	}
 }
 
-// ClaimMapToSlice returns an array of sorted exchange rate ballots.
-func ClaimMapToSlice(claims map[string]Claim) []Claim {
+// ClaimMapToSlices returns an array of sorted exchange rate ballots and a second
+// array woth validators not in the includeMap filtered out.
```

Comment:
```suggestion
// ClaimMapToSlices returns an array of sorted exchange rate ballots and a second
// array with validators not in the includeMap filtered out.
```

Let's also describe why we do the filtering